src/quacc/calculators/orca/orca.py

```diff
@@ -0,0 +1,193 @@
+"""
+A wrapper around ASE's ORCA calculator that makes it better suited for
+high-throughput DFT with multi-level schemes.
+"""
+
+from __future__ import annotations
+
+import numpy as np
+from ase import Atoms
+from ase.calculators.orca import ORCA as ORCA_
+from ase.calculators.orca import OrcaTemplate
+from ase.utils import writer
+
+
+class ORCA(ORCA_):
+    """
+    This is a wrapper around the ASE ORCA calculator that allows retains all the
+    functionality of the original ORCA calculator whilst adding the ability to
+    make dangling bonds of pseudo-hydrogens if the atoms object contains the arrays
+    "oxi_states" and "atom_type".
+
+    Returns
+    -------
+    Atoms
+        The ASE Atoms object with attached ORCA calculator.
```

Comment:
Parameter details go below the `__init__`. See https://github.com/Quantum-Accelerators/quacc/blob/main/src/quacc/calculators/vasp/vasp.py for an example. An `Atoms` object is also not what's returned.